sidekick_core/lib/src/commands/plugins/create_templates/shared_code.dart

```diff
@@ -64,22 +68,29 @@ Future<void> main() async {
   }
   pubGet(package);
   
-  final commandFile = package.root.file('lib/src/${pluginName.snakeCase}.dart');
-  commandFile.writeAsStringSync("""
-$exampleCommand
-""");
+  // Platform.script.path -> <plugin>/tool/install.dart
+  final pluginDir = File(Platform.script.path).parent.parent;
```

Comment:
Could we hide this "hack" in a `PluginContext` API? It would allow us to eventually change the implementation and hide the manual dir traversal.